arch/arm/soc/st_stm32/stm32f4/Kconfig.defconfig.series

```diff
@@ -32,4 +32,11 @@ config I2C_STM32_V1
 
 endif # I2C_STM32
 
+if RTC
```

Comment:
Since it will likely be common for all series, why not moving this in 
arch/arm/soc/st_stm32/common/Kconfig.defconfig.series ?

Reply:
Done.